CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ### ...
 
+### Enhancements
+#### ([1061](https://github.com/allegro/hermes/pull/1061)) Introduced HdrHistogram-based metrics reservoir
+
+Introduced [HdrHistogram](http://hdrhistogram.org/)–based implementation of reservoir in metrics.
+To switch between implementations, use `metrics.reservoir.type` option:
+
+* `exponentially_decaying` - exponentially decaying reservoir (previous version, currently default)
+* `hdr` - HdrHistogram–based reservoir 
+
```

Comment:
I would add an information why it’s worth to consider using hdr histogram. Something like: 'HdrHistogram-based reservoir has much less memory footprint than exponentially decaying. Because of this GC in Hermes has less job to do which leads to better Hermes performance. In the near future HDR will be default Hermes metrics reservoir.